dev/com.ibm.ws.jaxrs.2.1.sse_fat/fat/src/com/ibm/ws/jaxrs21/sse/fat/BasicSseTest.java

```diff
@@ -83,4 +83,5 @@ public void testSseWithRX() throws Exception {
     public void testErrorSse() throws Exception {
         runTest(server, SERVLET_PATH, "testErrorSse");
     }
+    
```

Comment:
I think if you remove this blank line, then you wouldn't need to change this file at all.

Reply:
Done